azure/durable_functions/models/DurableOrchestrationStatus.py

```diff
@@ -26,7 +27,9 @@ def __init__(self, name: Optional[str] = None, instanceId: Optional[str] = None,
             if lastUpdatedTime is not None else None
         self._input: Any = input
         self._output: Any = output
-        self._runtime_status: Optional[str] = runtimeStatus  # TODO: GH issue 178
+        self._runtime_status: Optional[OrchestrationRuntimeStatus] = runtimeStatus
+        if runtimeStatus is not None:
+            self._runtime_status = OrchestrationRuntimeStatus(runtimeStatus)
```

Comment:
I'm confused as to the point of this wrapper. Isn't `runtimeStatus` already an `Optional[OrchestrationRuntimeStatus]` type already?

Reply:
It wasn't, as described in one of our threads here: https://github.com/Azure/azure-functions-durable-python/pull/164
Below is a pic of the relevant context, which I'll wrap in delimiters because it looks just like the GitHub UI 😄 :

**Context Pic start:**
![connor_source](https://user-images.githubusercontent.com/9623336/92662743-560dc700-f2b4-11ea-83c3-23698dc0c496.PNG)
**Context Pic End:**

So this PR is all about resolving that typechecking mismatch I mentioned in that earlier thread